app/javascript/packs/ping_thing/index_template.vue

```diff
@@ -0,0 +1,128 @@
+    </div>
+    <div class="card mb-4">
+      <div class="card-content">
+        <h3 class="card-heading mb-4">{{ network }} TX Confirmation Time</h3>
+        <scatter-chart :vector="ping_things" fill_color="rgba(221, 154, 229, 0.4)" line_color="rgb(221, 154, 229)"/>
+      </div>
+    </div>
+
+    <div class="card">
+      <div class="table-responsive-lg">
+        <table class='table mb-0'>
+          <thead>
+          <tr>
+            <th class="align-middle column-md-sm">Success / Time</th>
+            <th class="align-middle column-lg">
+              Reported&nbsp;At<br />
+              <span class="small text-muted">Signature</span>
+            </th>
+            <th class="align-middle column-lg">
+              Application<br />
+              <span class="small text-muted">Type (Commitment Level)</span>
+            </th>
+            <th class="align-middle column-xs">Posted&nbsp;By</th>
+          </tr>
+          </thead>
+
+          <tbody>
+            <tr v-for="(pt, index) in ping_things" :key="pt.id">
+              <td class="align-middle">
+                <span v-html="success_icon(pt.success)"></span>
+                <strong class="text-success h6">{{ pt.response_time.toLocaleString() }}</strong>&nbsp;ms
+              </td>
+              <td class="align-middle small">
+                {{ pt.reported_at }}<br />
+                <span class="word-break">
+                  <a :href="link_from_signature(pt.signature)" target="_blank" class="small">
+                    {{ pt.signature.substring(0,6) + "..." + pt.signature.substring(pt.signature.length - 4, pt.signature.length) }}
+                  </a>
+                </span>
+              </td>
+              <td class="align-middle small">
+                <div v-if="pt.application">
+                  {{ pt.application }}<br />
+                </div>
+                <span class="text-muted">
+                  <span v-html="transaction_type_icon(pt.transaction_type)"></span>
+                  {{ pt.transaction_type }}
+                  <span v-if="pt.commitment_level">
+                    ({{ pt.commitment_level }})
+                  </span>
+                </span>
+              </td>
+              <td class="align-middle text-muted">{{ pt.username }}</td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+  import axios from 'axios'
+  import loadingImage from 'loading.gif'
+  import scatterChart from './scatter_chart'
+
+  axios.defaults.headers.get["Authorization"] = window.api_authorization
+
+  export default {
+    props: ['network'],
+    data () {
+      return {
+        ping_things: []
+      }
+    },
+    created () {
+      var ctx = this
+      axios.get("/api/v1/ping-thing/testnet")
+           .then(function(response){
+             ctx.ping_things = response.data
+           })
+    },
+    methods: {
+      success_icon(success) {
```

Comment:
Please remove related ruby method that is no longer needed, to keep the code clean.